src/patternfly/layouts/Page/styles.scss

```diff
@@ -0,0 +1,73 @@
+// URL.com/guidelines#layout
+@import "../../patternfly-utilities";
+
+$pf-l-page__sidebar-Width:    15rem !default;
+$pf-l-page--breakpoint:       $pf-global--breakpoint--sm !default;
+
+.pf-l-page {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
```

Comment:
Do we want to wrap it? I don't know, that will depend on how we deal with the header. We can leave it s is for now and change it when we create the page demo